src/EVM/Stepper.hs

```diff
@@ -22,10 +23,12 @@ where
 -- as the framework for monadic interpretation.
 
 import Control.Monad.IO.Class
-import Control.Monad.Operational (Program, ProgramViewT(..), ProgramView, singleton, view)
+import Control.Monad.Operational (Program, ProgramViewT(..), ProgramView, singleton)
 import Control.Monad.ST (stToIO, RealWorld)
-import Control.Monad.State.Strict (execStateT, runStateT, get)
+import Control.Monad.State.Strict (execStateT, get)
 import Data.Text (Text)
+import Control.Monad.State.Strict (StateT(..))
+import Control.Monad.Operational qualified as Operational
```

Comment:
Can we merge these two with the same imports above?

Reply:
Good point! Fixed :)